src/packages/board/board.py

```diff
@@ -124,13 +130,24 @@ def __init__(
         Modification lock to prevent concurrent modification of the board.
         """
 
+        self._recently_closed: typing.Deque[Rescue] = collections.deque(maxlen=max_recently_closed)
+        """ Recently closed rescue deck """
+        self._offline_rescue_storage: typing.Deque[Rescue] = collections.deque()
+        """ Holds closed cases that couldn't be pushed to the API (probably due to offline mode) """
+
         super(RatBoard, self).__init__()
 
+    @property
+    def recently_closed(self) -> typing.Deque[Rescue]:
+        """ Recently closed rescues """
+        return self._recently_closed
+
     async def on_online(self):
         logger.info("Rescue board online.")
         self._offline = False
-        # TODO get API version from remote and log it
-        # TODO emit canned offline events to API
+        logger.info("emitting {} cached rescues to the API...", len(self._offline_rescue_storage))
+        for rescue in self._offline_rescue_storage:
+            await self._handler.update_rescue(rescue)
```

Comment:
[Error] Shouldn't this be using `self._offline_rescue_storage.popleft()`?
As it stands, nothing is emptying the deque, meaning if we went off/online again, rescues would be sent again.

Reply:
absolutely correct, fixed.